### PR TITLE
Tweak: Advanced Pirate Shipyards

### PR DESCRIPTION
**Pull request by Zitchas**
### Issue: 
- Some people on discord commented that - despite the planet description implying that Buccaneer Bay is one of the most prosperous and advanced pirate shipyards and mentioning that it produces pirated republic and syndicate ships, it doesn't carry some of the most common pirate ships. 

### Fix Details:
Creates the shipyard, with the following ships:
- Quicksilver
- Splinter
- Wasp

This fixes a discrepancy with the planet description, wherein it states that the shipyard carries pirated "Republic and Syndicate" ships; but doesn't carry some very common pirate ships from those shipyards.

### Impact:
While including the Quicksilver and Wasp aren't likely to affect the player's ease of access to these very much as they are already available at 8 other ports; the splinter is only available at two. All of those ports are likely to be hard to access for players engaged in piracy. For these players, this PR gives some means of purchasing these ships. For players that haven't angered the Syndicate, this PR is unlikely to affect the availability or usefulness of these ships.

### CODE REVIEW

**Comment by Amazinite** [comment]
How about also having them sell Vanguards too? Ref: #4293
And while we're here, might be worth looking at Smuggler's Den as well, since its description similarly promises special access to ships, but doesn't actually provide anything out of the ordinary, aside from perhaps the Mod Argosy. The south is a deadzone for AM turrets when the Navy largely uses missiles during the campaign, so might be worth selling AM turrets on Smuggler's Den (and updating the description slightly to also mention it selling outfits one normally wouldn't be able to find in the region).
image

**Comment by Zitchas** [comment]
Sure, I can do that. As far as that goes, there was talk about having some marauders for sale at some pirate ports. Should I include those? Or just add the vanguard and anti-missile and leave it at that?

**Comment by Amazinite** [comment]
The sale of Marauder ships seems like a topic for another time.

**Comment by beccabunny** [comment]
There's also Haven, in the far north, which claims to "have a shipyard where customized ships are sold" but sells only the normal Berserker, Leviathan and Firebird

**Comment by Amazinite** [comment]
Might be worth changing the Haven and Smuggler's Den descriptions to say that the modified ships aren't for sale, but you need connections to get them, opening the possibility of gaining Marauder ships through a storyline instead of having incorrect descriptions.

**Comment by Zitchas** [comment]
In regards to the AM, I simply added the "Lovelace Basics" outfitter to Smuggler's Den. Does that work, or should I make an AM-specific outfitter that just gives access to that? 

(just to save time: )
```
outfitter "Lovelace Basics"
	"Anti-Missile Turret"
	"Sidewinder Missile"
	"Sidewinder Missile Launcher"
	"Sidewinder Missile Rack"
	"D23-QP Shield Generator"
	"Torpedo"
	"Cargo Scanner"
	"Luxury Accommodations"
```

I think that's all the changes suggested. How does it look now?

File: data/map.txt [hunk 1]
```diff
@@ -26671,12 +26672,14 @@ planet Slylandro
...
 	landscape land/loc1
-	description `Smuggler's Den was formerly known as Grendel Station, back when it was an operating refinery. But, it has since been taken over by pirates, who turned it into a haven for all manner of smugglers and privateers. The repair shops here can make a variety of legal and illegal modifications to your ship, and in the shipyard you can buy refurbished starships... if it does not trouble your conscience to think about the likely fates of their previous owners.`
+	description `Smuggler's Den was formerly known as Grendel Station, back when it was an operating refinery. But, it has since been taken over by pirates, who turned it into a haven for all manner of smugglers and privateers. Being a hub of pirate activity, Smuggler's Den has access to fenced goods from beyond the region. The repair shops here can make a variety of legal and illegal modifications to your ship, and in the shipyard you can buy refurbished starships... if it does not trouble your conscience to think about the likely fates of their previous owners. Rumor has it that pilots with the right connections may be able to purchase heavily modified ships here.`
```
Comment (Amazinite) [inline data/map.txt]: Fenced goods?

Comment (tehhowch) [reply data/map.txt]: That's fine (stolen goods). `Being a hub` is better phrased `As a hub` IMO.

I would probably also join that sentence with the bit about making modifications, and start a new sentence regarding the ships for sale.

Comment (Zitchas) [reply data/map.txt]: Made both suggested changes. How does this sound?

`As a hub of pirate activity, Smuggler's Den has access to fenced goods from beyond the region, resulting in repair shops that can make a variety of legal and illegal modifications to your ship. This confluence also provides the opportunity to buy refurbished starships... if it does not trouble your conscience to think about the likely fates of their previous owners.`

**Comment by Hyugat** [comment]
vanguards are never once used by pirate fleets, to have them for sale here does not make sense. Changes are good other than that.

**Comment by Zitchas** [comment]
I think Amazinite suggested including the Vanguard at these two "higher end" shipyards as there was a PR to add them (and some other higher-end civilian ships) to pirate fleets in order to increase diversity, which is a good thing in my opinion.

**Comment by Amazinite** [comment]
And while random pirate fleets don’t currently use Vanguards, some bounty fleets do.

**Comment by Amazinite** [comment]
How about instead of "Advanced Pirate", we give each region an advanced shipyard.
```
Northern Pirate
	Dagger
	Berserker
	Headhunter
	Firebird

Advanced Northern Pirate
	Aerie
	Mule
	Leviathan

Core Pirate
	Wasp
	Quicksilver
	Splinter
	Manta

Advanced Core Pirate
	Protector
	Vanguard

Southern Pirate
	Hawk
	Fury
	Clipper
	Argosy

Advanced Southern Pirate
	Modified Argosy
	Osprey
	Falcon
```

Then in terms of what pirate planet gets what:
```
Greenrock
	Northern
	Core
	Southern
	Advanced Southern

Smuggler's Den
	Southern
	Core
	Advanced Southern
	Advanced Core

Covert
	Southern
	Core

Buccaneer Bay
	Southern
	Core
	Northern
	Advanced Core

Freedom
	Same as right now

Haven
	Core
	Northern
	Advanced Northern
```

**Comment by Zitchas** [comment]
That distribution looks good to me. It feels like more diversity among the various pirate ports. Also sets it up nicely for adding future pirate ports with their own varieties, if something like the Pirate Campaign should happen to need that functionality.

**Pull request merged**